Rubberduck.Core/UI/Controls/EmptyUIRefresh.xaml

```diff
@@ -8,7 +8,7 @@
 
     <Border Background="AliceBlue" DataContext="{Binding RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type UserControl}}, Path=DataContext}">
         <TextBlock VerticalAlignment="Center" HorizontalAlignment="Center" TextWrapping="WrapWithOverflow" TextAlignment="Center"
-                       MinWidth="200" Visibility="{Binding EmptyUIRefreshMessageVisibility}">
```

Comment:
Why delete?

Reply:
I reversed the logic of how the empty UI refresh is displayed, so that binding no longer exists.  I was also thinking that the MinWidth was impossible to violate in the current layout, but I'll verify that.